test/functional/p2p_ibd_txrelay.py

```diff
@@ -22,10 +25,12 @@ def set_test_params(self):
             ["-minrelaytxfee={}".format(NORMAL_FEE_FILTER)],
             ["-minrelaytxfee={}".format(NORMAL_FEE_FILTER)],
         ]
+
     def run_test(self):
         self.log.info("Check that nodes set minfilter to MAX_MONEY while still in IBD")
         for node in self.nodes:
             assert node.getblockchaininfo()['initialblockdownload']
+            wait_until(lambda: (conn_info['minfeefilter'] != 0 for conn_info in node.getpeerinfo()))
             for conn_info in node.getpeerinfo():
                 assert_equal(conn_info['minfeefilter'], MAX_FEE_FILTER)
```

Comment:
Could also replace the above 3 lines with the following, but it wouldn't be testing exactly the same thing.
```python
wait_until(lambda: (conn_info['minfeefilter'] == MAX_FEE_FILTER for conn_info in node.getpeerinfo()))
```

Reply:
A tuple with more than one element always evaluates to true, so your exact suggestion wouldn't work, but Concept ACK replacing this with just `wait_until`

Reply:
Thank you -- your comment made me realise that there is Python `any(iterable)` like the Ruby [Enumerator#all?](https://ruby-doc.org/core-2.7.1/Enumerable.html#method-i-all-3F) I was familiar with. Mimicked the way it is used in `test/functional/test_framework/util.py:443-444` and applied to both assertions for symmetry and safety.

Reply:
That also allowed no longer importing anything from `test_framework.util` :+1: